Dockerfile

```diff
@@ -338,17 +337,10 @@ RUN systemctl disable test-integration-ipfs-offline
 VOLUME /home/rootless/.local/share
 COPY ./Dockerfile.d/test-integration-rootless.sh /
 RUN chmod a+rx /test-integration-rootless.sh
-CMD ["/test-integration-rootless.sh", \
```

Comment:
Why? This makes it harder to run tests on a laptop

Reply:
I can restore a default CMD.